### DISCUSSION TITLE
v3.3.2

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#1207] [Announcements]
## What's Changed

* Fix nan being serialized (fixes #1038) by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/1094
* Add missing return in BleFingerprint::fingerprintServiceAdvertisements / Fix incorrect check in hextostr() by @gunnarbeutner in https://github.com/ESPresense/ESPresense/pull/1097
* Add mqtt command skip_ms and skip_distance by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/1109
* Copy all statistics values when a device's MAC address changes by @gunnarbeutner in https://github.com/ESPresense/ESPresense/pull/1098
* Fix race condition by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/1114
* Improve 1-Euro distance filtering, add spike filter by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/1108
* Send events by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/1120
* Add variance reporting by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/1135
* Use default advertising intervals to fix high variance by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/1139
* Add build timestamp to the version number by @gunnarbeutner in https://github.com/ESPresense/ESPresense/pull/1145
* Add https:// -> wss:// replace call by @scottt732 in https://github.com/ESPresense/ESPresense/pull/1156
* miTherm: temperature & humidity decimal precision set to 2 by @lolorc in https://github.com/ESPresense/ESPresense/pull/1154
* Bump actions/setup-python from 4 to 5 by @dependabot in https://github.com/ESPresense/ESPresense/pull/1160
* Sensor DS18B20  Check to Skip Null Values by @creuzerm in https://github.com/ESPresense/ESPresense/pull/1172
* Add support for EST-PoE-32 by @EverythingSmartHome in https://github.com/ESPresense/ESPresense/pull/1186
* Increase internal time by @arne182 in https://github.com/ESPresense/ESPresense/pull/1163
* SGP30: Corrected measure interval, added separate Reporting Interval, Corrected TVOC device class by @FvanEgmond in https://github.com/ESPresense/ESPresense/pull/1203
* Bump AsyncWiFiSettings to 1.0.10 (fixes #1192) by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/1206

## New Contributors
* @scottt732 made their first contribution in https://github.com/ESPresense/ESPresense/pull/1156
* @lolorc made their first contribution in https://github.com/ESPresense/ESPresense/pull/1154
* @EverythingSmartHome made their first contribution in https://github.com/ESPresense/ESPresense/pull/1186
* @arne182 made their first contribution in https://github.com/ESPresense/ESPresense/pull/1163
* @FvanEgmond made their first contribution in https://github.com/ESPresense/ESPresense/pull/1203

**Full Changelog**: https://github.com/ESPresense/ESPresense/compare/v3.3.1...v3.3.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v3.3.2'>v3.3.2</a>.</em>

**Comment by jkpe** [comment]
Moving between base stations with an iPhone seems much faster in this version, thanks very much!
